ci-operator/step-registry/ipi/conf/azure/oidc-creds-provision/ipi-conf-azure-oidc-creds-provision-commands.sh

```diff
@@ -149,7 +150,16 @@ ccoctl azure create-all \
   --dnszone-resource-group-name="${BASE_DOMAIN_RESOURCE_GROUP_NAME}" \
   --storage-account-name="$(tr -d '-' <<< ${CLUSTER_NAME})oidc" \
   --output-dir="/tmp" \
-  ${ADDITIONAL_CCOCTL_ARGS}
+  ${ADDITIONAL_CCOCTL_ARGS} &> "${ccoctl_ouptut}"
+cat "${ccoctl_ouptut}"
+
+# get oidc_provider_issuer_url from `oc get Authentication cluster -o json`, so not have to enable the follwoing lines yet 
+# save oidc_provider info for upgrade
+#oidc_provider_issuer_url=$(grep "Issuer URL .* is" "${ccoctl_ouptut}" | awk -F"is " '{print $NF}')
+#if [[ -n "${oidc_provider_issuer_url}" ]]; then
+#  echo "Saving oidc_provider_arn: ${oidc_provider_issuer_url}"
+#  echo "${oidc_provider_issuer_url}" > "${SHARED_DIR}/azure_oidc_provider_issuer_url"
+#fi
```

Comment:
here we annotate azure and gcp by purpose?

Reply:
Right, because I checked the oidc provision output, we can got the info from cluster install, or some others.
For Azure, it is https://github.com/openshift/release/pull/43589/files#diff-10a5c92def4ce722760b3d053d7b5c5ef546580ea75ef49bb81b9d1374e2fe22R156
For GCP, it is https://github.com/openshift/release/pull/43589/files#diff-84ed1b07284a984b67521516323d4dfe336646c84354f08bc61e99b3e028c5e2R74

Reply:
ack, i will follow above notes to get the info when adding the steps for azure/gcp. thanks